docs/user-guide/catalog.md

```diff
@@ -199,6 +205,8 @@ In addition to the standard options, only for WMS catalog sources, through the *
 
     - Produce a localized output for GetFeatureInfo requests (the freemarker template need to be properly configured to retrieve [the locale from the request](https://docs.geoserver.org/stable/en/user/tutorials/freemarker.html))
 
+* *Set Visibility Limit*: this can be enabled when the user wants to automatically applies of the visibility limits suggested by the server.
```

Comment:
* *Set Visibility Limit*: available only for WMS layers coming from CSW or WMS catalog sources type. If checked and scale limits present in the WMS Capabilities (eg. MinScaleDenominator and/or MaxScaleDenominator), these will be automatically applied to the layer settings when a layer is added to the map from this source.